providers/src/airflow/providers/google/cloud/hooks/kubernetes_engine.py

```diff
@@ -434,38 +433,9 @@ def get_conn(self) -> client.ApiClient:
             enable_tcp_keepalive=self.enable_tcp_keepalive,
         ).get_conn()
 
-    def check_kueue_deployment_running(self, name, namespace):
```

Comment:
@moiseenkov why did we not deprecate this method?

Reply:
It was moved to the `KubernetesHook` and the `GKEKubernetesHook` inherits this method.